### PR TITLE
RavenDB-13278 : revision configuration options for collections should be case insensitive

### PR DESCRIPTION
**Pull request by aviv86**
https://issues.hibernatingrhinos.com/issue/RavenDB-13278

### CODE REVIEW

**Review by grisha-kotler — APPROVED** [review]

File: src/Raven.Server/Documents/Handlers/RevisionsHandler.cs [hunk 1]
```diff
@@ -61,7 +62,29 @@ public Task GetRevisionsConfig()
         [RavenAction("/databases/*/admin/revisions/config", "POST", AuthorizationStatus.DatabaseAdmin)]
         public async Task ConfigRevisions()
         {
-            await DatabaseConfigurations(ServerStore.ModifyDatabaseRevisions, "read-revisions-config");
+            await DatabaseConfigurations(
+                ServerStore.ModifyDatabaseRevisions,
+                "read-revisions-config",
+                beforeSetupConfiguration: (string name, ref BlittableJsonReaderObject configuration, JsonOperationContext context) =>
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/RevisionsHandler.cs:68]: I'm assuming that this happens before we go to the cluster state machine, right?

Comment (ayende) [reply src/Raven.Server/Documents/Handlers/RevisionsHandler.cs:68]: What happens if we have two separate requests like this? Is this a whole sale replacement over time, or individual ops?

Comment (ppekrol) [reply src/Raven.Server/Documents/Handlers/RevisionsHandler.cs:68]: Full replacement: https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Server/ServerWide/Commands/EditRevisionsConfigurationCommand.cs#L28

Comment (aviv86) [reply src/Raven.Server/Documents/Handlers/RevisionsHandler.cs:68]: yes,  this happens before we go to the cluster state machine

**Review by ayende — APPROVED** [review]

**Pull request merged**